apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -630,6 +630,74 @@ cm_publish_send(Peer, Solution) ->
 		body => ar_serialize:jsonify({Json})
 	})).
 
+%% @doc Fetch the jobs from the pool proxy or coordinated mining exit peer.
+get_jobs(PeerOrURL, PrevOutput, Pool) ->
+	{Peer, Headers, BasePath} =
+		case Pool of
+			false ->
+				{PeerOrURL, cm_p2p_headers(), ""};
+			true ->
+				#{ host := Host, port := Port, path := P } = uri_string:parse(PeerOrURL),
+				{{binary_to_list(Host), Port}, pool_client_headers(), binary_to_list(P)}
+		end,
+	handle_get_jobs_response(ar_http:req(#{
+		peer => Peer,
+		method => get,
+		path => BasePath ++ "/jobs/" ++ binary_to_list(ar_util:encode(PrevOutput)),
```

Comment:
PrevOutput "" effectively makes call to /jobs, and that endpoint is missing

Reply:
But actually binary_to_list is not correct
There should be pattern matching for path
Because if http://host:123, then path is <<>>
But if http://host:123/ then path is "/"
But binary_to_list("/") causes bad argument